manifests/server.pp

```diff
@@ -16,7 +16,7 @@
   Hash $databases                          = {},
   $ldap_ifs                                = ['/'],
   $ldaps_ifs                               = [],
-  $ldapi_ifs                               = ['/'],
+  $ldapi_ifs                               = undef,
```

Comment:
Maybe I didn't follow the whole thread, but why is the default changed?

Reply:
Sorry I didn't cleaned this part from the other PR when I splitted the original PR...

Reply:
Now it's removed.

Reply:
ok